src/SIL.XForge.Scripture/ClientApp/src/app/my-projects/my-projects.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, OnInit } from '@angular/core';
+import { translate } from '@ngneat/transloco';
+import { isPTUser } from 'realtime-server/lib/esm/common/models/user';
+import { isResource } from 'realtime-server/lib/esm/scriptureforge/models/sf-project';
+import { UserDoc } from 'xforge-common/models/user-doc';
+import { OnlineStatusService } from 'xforge-common/online-status.service';
+import { SubscriptionDisposable } from 'xforge-common/subscription-disposable';
+import { SFUserProjectsService } from 'xforge-common/user-projects.service';
+import { UserService } from 'xforge-common/user.service';
+import { environment } from '../../environments/environment';
+import { ParatextProject } from '../core/models/paratext-project';
+import { SFProjectProfileDoc } from '../core/models/sf-project-profile-doc';
+import { ParatextService } from '../core/paratext.service';
+
+/** Presents user with list of available projects to open or connect to. */
+@Component({
+  selector: 'app-my-projects',
+  templateUrl: './my-projects.component.html',
+  styleUrls: ['./my-projects.component.scss']
+})
+export class MyProjectsComponent extends SubscriptionDisposable implements OnInit {
+  /** SF projects that the current user is on at SF. */
+  userConnectedProjects: SFProjectProfileDoc[] = [];
+  /** Resources on SF that the current user is on at SF. */
+  userConnectedResources: SFProjectProfileDoc[] = [];
+  /** PT projects that the user can access that they are not connected to on SF. */
+  userUnconnectedParatextProjects: ParatextProject[] = [];
+  user?: UserDoc;
+  problemGettingPTProjects: boolean = false;
+  loadingPTProjects: boolean = false;
+  initialLoadingSFProjects: boolean = true;
+  userIsPTUser: boolean = false;
+
+  constructor(
+    private readonly userProjectsService: SFUserProjectsService,
+    readonly paratextService: ParatextService,
+    private readonly onlineStatusService: OnlineStatusService,
+    private readonly userService: UserService
+  ) {
+    super();
+  }
+
+  /** If we are aware of any SF or PT projects that the user can access. */
+  get userHasProjects(): boolean {
+    return (
+      this.userConnectedProjects.length > 0 ||
+      this.userConnectedResources.length > 0 ||
+      this.userUnconnectedParatextProjects.length > 0
+    );
+  }
+
+  get isOnline(): boolean {
+    return this.onlineStatusService.isOnline;
+  }
+
+  async ngOnInit(): Promise<void> {
+    this.subscribe(this.userProjectsService.projectDocs$, (projects?: SFProjectProfileDoc[]) => {
+      if (projects == null) return;
+      this.userConnectedProjects = projects.filter(project => project.data != null && !isResource(project.data));
+      this.userConnectedResources = projects.filter(project => project.data != null && isResource(project.data));
+      this.initialLoadingSFProjects = false;
+    });
+    await this.loadUser();
+    this.subscribe(this.onlineStatusService.onlineStatus$, online => {
```

Comment:
Is there a reason you're subscribing to this, rather than doing something like:
``` ts
this.onlineStatusService.online.then(...)
```
If you use network tools to go online and offline (back and forth) you'll see the project reload every time. There doesn't seem to me to be a real reason to do that.